index.html

```diff
@@ -35,14 +35,236 @@ <h1> <strong>I’m Mariane
+                </div>
+                <div class="Tech">
+                    <ul>
+                        <li class="html">html</li>
+                        <li class="css">css</li>
+                        <li class="java">javaScript</li>
+                    </ul>
+                </div>
+                <div class="Submission">
+                    <button type="submit">See Project</button>
+                </div>
+            </section>
+        </div>
+
+
+
+
+
+        <div class="box box-2">
+            <div class="snap1"></div>
+            <section class="body1">
+                <header class="Multi">
+                    <div class="title2">
+                        <h1>Multi-Post
+                        Stories</h1>
+                    </div>
+                    <div class="quali1">
+                        <ul>
+                            <li class="can">CANOPY</li>
+                            <li class="back">Back End Dev</li>
+                            <li class="back">2015</li>
+                        </ul>
+                    </div>
+                </header>
+                <div class="para1">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <div class="Tech">
+                    <ul>
+                        <li class="html">html</li>
+                        <li class="css">css</li>
+                        <li class="java">javaScript</li>
+                    </ul>
+                </div>
+                <div class="Submission">
+                    <button type="submit">See Project</button>
+                </div>
+            </section>
+        </div>
+
+
+
+
+
+
+
+
+        <div class="box box-3">
+            <div class="snap3"></div>
+            <section class="body">
+                <header class="Tonic">
+                    <div class="title1">
+                        <h2>Tonic</h2>
+                    </div>
+                    <div class="quali">
+                        <ul>
+                            <li class="can">CANOPY</li>
+                            <li class="back">Back End Dev</li>
+                            <li class="back">2015</li>
+                        </ul>
+                    </div>
+                </header>
+                <div class="para1">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <div class="Tech">
+                    <ul>
+                        <li class="html">html</li>
+                        <li class="css">css</li>
+                        <li class="java">javaScript</li>
+                    </ul>
+                </div>
+                <div class="Submission">
+                    <button type="submit">See Project</button>
+                </div>
+            </section>
```

Comment:
`Figma template`
![Screenshot from 2022-04-30 09-42-37](https://user-images.githubusercontent.com/77842237/166095018-3d827650-a75e-44ed-b809-ccc4b8f19c15.png)
`your work`
![Screenshot from 2022-04-30 09-42-37](https://user-images.githubusercontent.com/77842237/166095091-b6adf0aa-166b-45dd-9dc3-44928888c0a9.png)

- [ ] Kindly You must stick to the design as much as possible (e.g., font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).